release-content/0.16/release-notes/gpu_driven_rendering.md

```diff
@@ -0,0 +1,75 @@
+## GPU-Driven Rendering
+
+Over the years, the trend in real-time rendering has increasingly been to move work from the CPU to the GPU. One of the latest developments in this area has been *GPU-driven rendering*, in which the GPU takes a representation of the scene and essentially works out what to draw on its own. While Bevy 0.15 had some support for GPU-driven rendering for meshlets only, Bevy 0.16 features comprehensive support for this technique, including for skinned meshes. This dramatically reduces the amount of CPU time that the renderer needs for larger scenes. It's automatically enabled on platforms that support it; unless your application hooks into the rendering pipeline, upgrading to Bevy 0.16 will automatically enable GPU-driven rendering for your meshes.
+
+To explain how GPU-driven rendering operates, it's easiest to first describe how CPU-driven rendering works:
+
+1. The CPU determines the objects that are visible, via frustum culling and perhaps occlusion culling.
+
+2. For each such object:
+
+   a. The CPU sends the object's transform to the GPU, possibly in addition to other data such as joint weights.
+
+   b. The CPU tells the GPU where the mesh data is.
+
+   c. The CPU writes the material data to the GPU.
+
+   d. The CPU tells the GPU where textures and other buffers needed to render the objects are (light data, etc.)
+
+   e. The CPU issues a drawcall.
+
+   f. The GPU renders the object.
+
+In contrast, GPU-driven rendering in Bevy works like this:
+
+1. The CPU uploads transform information for all objects at once to the GPU.
```

Comment:
This is the same as for CPU-driven.

Reply:
I intentionally gloss over the difference between `MeshInputUniform` and `MeshUniform`, and the difference between the GPU-preprocessing and no-GPU-preprocessing paths, the details of which I thought were too complex and out-of-scope for this blog post.

I reworded it to "The CPU supplies a single buffer containing transform information for all objects to the GPU, so that shaders can process many objects at once.".